src/shared/api/OneSignalApiBase.ts

```diff
@@ -145,6 +136,23 @@ export class OneSignalApiBase {
       );
     }
   }
+
+  // OneSignal's backend requires that all request have a
+  // have a app_id in the UUID format in the request
+  private static requestHasAppId(
+    url: string,
+    body?: Record<string, unknown>,
+  ): boolean {
+    if (url.startsWith('apps/')) {
+      const parts = url.split('/');
+      return isValidUuid(parts[1]);
+    }
+
```

Comment:
Add comment, that the only thing using body `app_id` is the welcome notification.